app-emulation/wineasio/wineasio-9999.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2010 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+[[ "${PV}" = "9999" ]] && inherit subversion
+inherit multilib-minimal
+
+DESCRIPTION="ASIO driver for WINE"
+HOMEPAGE="http://sourceforge.net/projects/wineasio"
+if [ "${PV}" = "9999" ]; then
+	ESVN_REPO_URI="https://${PN}.svn.sourceforge.net/svnroot/${PN}/trunk/${PN}"
+	KEYWORDS=""
+else
+	SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+	KEYWORDS="~x86 ~amd64"
+fi
+RESTRICT="mirror"
+LICENSE="GPL-2"
+IUSE=""
+SLOT="0"
+
+DEPEND="media-libs/asio-sdk"
+RDEPEND=">=app-emulation/wine-0.9.35[${MULTILIB_USEDEP}]
+	>=media-sound/jack-audio-connection-kit-1.9.9.5[${MULTILIB_USEDEP}]"
+
+S="${WORKDIR}/${PN}"
+
+src_prepare() {
+	cp /opt/asiosdk2.3/ASIOSDK2.3/common/asio.h .
```

Comment:
You should be using a proper patch instead of this... ugly hack to include this header file; Or else, put it in 'FILESDIR'.